recipe/meta.yaml

```diff
@@ -1,49 +1,48 @@
-{% set version = "4.65.0" %}
+{% set version = "4.66.2" %}
 
 package:
   name: tqdm
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/t/tqdm/tqdm-{{ version }}.tar.gz
-  sha256: 1871fb68a86b8fb3b59ca4cdd3dcccbc7e6d613eeed31f4c332531977b89beb5
+  sha256: 6cd52cdf0fef0e0f543299cfc96fec90d7b8a7e88745f411ec33eb44d5ed3531
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - tqdm = tqdm.cli:main
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools_scm
+    - setuptools_scm >=3.4
     - setuptools
     - toml
     - wheel
   run:
     - python
     - colorama  # [win]
   run_constrained:
-    - slack-sdk
```

Comment:
a run_constrained entry without any constraints doesn't do anything https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained

Reply:
Thank you! That makes more sense especially regarding the importance of pinnings on optional dependencies. I am curious how, if at all, it affects our users. The original question was asked because I wondered in general, by removing them from the recipe, will our users notice a difference? If so, would they be better placed in the `run` section since they have no constraints?

Reply:
If we would place them in the run section, that would make them part of any installation of tqdm. run_constrained usage is, if any of these packages happen to be installed in the same environment, make sure they satisfy the constraint.

Reply:
Okay, that makes sense! We never included in the installation in the first place, so by removing them, there are no user-facing changes.